main/autogen/autogen.cc

```diff
@@ -314,6 +320,10 @@ class AutogenWalk {
         return tree;
     }
 
+    void setAutogenVersion(int version) {
+        autogenVersion = version;
+    }
```

Comment:
Can remove this setter and just have line 346 below be `walk.autogenVersion = autogenVersion` if that is more preferable and C++-y